docs/gettingstarted/configuration.rst

```diff
@@ -76,3 +76,74 @@ Environment variables
+
+
+``LANGUAGE_CODE``
+  Sets the language. Defaults to `en`.
+  Further `documentation <https://docs.djangoproject.com/en/2.1/ref/settings/#language-code>`_
+
+``USE_I18N``
+  Whether or not to enable Djangos translation system I18N. Defaults to `True`.
+  Further `documentation https://docs.djangoproject.com/en/2.1/ref/settings/#use-i18n>`_
+
+``USE_L10N``
+  Whether or not to use localization - eg date formatting. Defaults to `True`.
+  Further `documentation https://docs.djangoproject.com/en/2.1/ref/settings/#use-l10n>`_
+
+``USE_TZ``
+  Enable or disable timezone awareness. Defaults to `True`.
+  Further `documentation <https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-USE_TZ>`_
+
+Logging and handler levels:
+
+``ROOT_LOG_LEVEL`` - The 'root' log level, defaults to 'INFO'.
+``MAIL_ADMINS_HANDLER_LEVEL`` - The mail admin handler log level, defaults to `ERROR`.
+``CONSOLE_HANDLER_LEVEL`` - The console handler log level, defaults to `DEBUG`.
+``DJANGO_LOG_LEVEL`` - The django log level, defaults to `INFO`.
+``DJANGO_SERVER_LOG_LEVEL`` - The django.server log level, defaults to `INFO`.
+``SALEOR_LOG_LEVEL`` - The saleor log level, defaults to `DEBUG`.
+
+Further documentation on logging can be found
+`here <https://docs.djangoproject.com/en/2.1/topics/logging/#module-django.utils.log>`_
+
+``AVAILABLE_CURRENCIES``
+  A comma separated list of the currencies to be available for selection.
+
+``ACCOUNT_ACTIVATION_DAYS``
+  Defaults to `3`.
+
+``LOGIN_REDIRECT_URL``
+  The relative url to redirect a user after unsuccessful login attempts. Defaults to `home`.
+
+``LOW_STOCK_THRESHOLD``
+  The level of stock for a product line under which it will be marked as low stock. Defaults to `10`.
+
+``MAX_CART_LINE_QUANTITY``
+  Maximum number of items of a product line allowed to be added to the cart. Defaults to `50`.
+
+``PAGINATE_BY``
+  Number of products per page to display in the store front. Defaults to `16`.
+
+``DASHBOARD_PAGINATE_BY``
+  Number of products per page to display in the admin dashboard. Defaults to `30`.
+
+``DASHBOARD_SEARCH_LIMIT``
+  Number of search results to display after a product search. Defaults to `5`.
+
+Placeholder images:
+``PLACEHOLDER_IMAGE_60`` - The relative path of the 60 x 60 placeholder image. Defaults to `images/placeholder60x60.png`.
+``PLACEHOLDER_IMAGE_120`` - The relative path of the 120 x 120 placeholder image. Defaults to `images/placeholder120x120.png`.
+``PLACEHOLDER_IMAGE_255`` - The relative path of the 255 x 255 placeholder image. Defaults to `images/placeholder255x255.png`.
+``PLACEHOLDER_IMAGE_540`` - The relative path of the 540 x 540 placeholder image. Defaults to `images/placeholder540x540.png`.
+``PLACEHOLDER_IMAGE_1080`` - The relative path of the 1080 x 1080 placeholder image. Defaults to `images/placeholder1080x1080.png`.
+``DEFAULT_PLACEHOLDER`` - The default placeholder value. Defaults to the `PLACEHOLDER_IMAGE_255` value.
```

Comment:
This you test the above lines results? Seems broken. Also inline code is a double grave accent instead of a single one in RST (` ``something`` `).

> @NyanKiyoshi -- You probably could pack `60x60, 120x120, ..., 1080x1080` together instead of repeating the same sentence and also precise it can be an absolute path?

> @stephenmoloney -- good suggestion, can you paste some idea exactly what you mean though ?

I was thinking of a table, maybe. (Sorry for the late answer, I'm swamped by work). Something like that:

---

You can customize the placeholder images, which come to the following dimensions:


| environment variable name | dimensions | default value |
| -- | -- | -- |
| `PLACEHOLDER_IMAGE_60` | 60x60 | `images/placeholder60x60.png`
| `PLACEHOLDER_IMAGE_120` | 120x120 | `images/placeholder120x120.png`
| `PLACEHOLDER_IMAGE_255` | 255x255 | `images/placeholder255x255.png`
| `PLACEHOLDER_IMAGE_540` | 540x540 | `images/placeholder540x540.png`
| `PLACEHOLDER_IMAGE_1080` | 1080x1080 | `images/placeholder1080x1080.png`

Reply:
Good suggestion, thanks.

Reply:
Why is it useful to override the paths to placeholders instead of providing custom images under the same path?

Reply:
The benefit of this change is minimal no doubt but providing this flexibility does no harm really. 
Overriding is optional, the default is to use the images under the same path.
For example, perhaps you want to keep less divergence of your fork with upstream.

Reply:
It does have potential impact of exceeding the maximum allowed environment size. For example Amazon Elastic Beanstalk has a total environment limit set to 4kB. Anything in excess is silently dropped from the environment configuration.

Reply:
Ok, I'll remove this block of code and rebase.